app/views/emergency/newCookieIssued.scala.html

```diff
@@ -0,0 +1,23 @@
+@()
+<html>
+<head>
+    <title>
+        New cookie link request
+    </title>
+</head>
+<body>
+<h1>Request a link to obtain a new cookie</h1>
+
+<form action="/emergency/send-cookie-link" method="post">
+    <div>
+        <label for="email">Enter your email:</label>
+        <input type="text" id="email" name="email"/>
```

Comment:
So similar to my point above it might be worth creating firstname and lastname fields and then state this will be sent to their @guardian.co.uk address.